build/setup-kubectl.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 DEFAULT_INSTALL_LOCATION="/app/kubectl/linux"
-DEFAULT_KUBECTL_VERSION="v1.21.2"
+DEFAULT_KUBECTL_VERSION="v1.25.5"
```

Comment:
This requires testing routing manager which uses this kubectl. I will send the request to Joyer to test it.